src/main/fc/gps_lap_timer.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Betaflight. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+#include <math.h>
+
+#include "platform.h"
+
+#ifdef USE_GPS_LAP_TIMER
+
```

Comment:
```suggestion

#include "common/maths.h"

```

Reply:
It seems like platform.h is needed to pass the automated testing. Why do you suggest removing it?
Also, just curious why isn't the #ifdef isn't needed here?

Reply:
@KarateBrot

Reply:
No, I just suggested to add `#include "common/maths.h"` so that you can get rid of the `MIN` and `MAX` macros down below (because both macros are defined in `maths.h`)